lib/ansible/modules/network/nxos/nxos_banner.py

```diff
@@ -93,7 +93,7 @@
 
 
 def execute_show_command(module, command):
-    format = 'json'
+    format = 'text'
```

Comment:
since the module is using `text` format, can you please remove `check_rc=retry_json` https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/nxos/nxos_banner.py#L101?

Also does the module need this anymore https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/nxos/nxos_banner.py#L128-L139? I think output is only `dict` type when format is `json`.

Reply:
ack. thanks